### ISSUE TITLE
should `normal_init` be enhanced to accept different functions?

### ISSUE THREAD

**Issue by yiheng-wang-nv** [Project-MONAI/MONAI#3827]
As described in https://github.com/Project-MONAI/tutorials/issues/558, we may need to enhance `normal_init`

**Comment by yiheng-wang-nv** [comment]
I just found that both`torch.nn.init.kaiming_normal_` and `torch.nn.init.xavier_normal_` use `mean=0` for the initialization. Therefore, now I'm not sure if it is reasonable to modify `normal_init`, which assigned different `mean` for different modules (0 for conv and linear modules, 1 for batch norm modules).
Making `normal_init` more versatile may totally change this function, thus so far let me just close this issue.
